utils/keycodes.js

```diff
@@ -7,3 +7,4 @@ export const UP = 38;
 export const RIGHT = 39;
 export const DOWN = 40;
 export const DELETE = 46;
+export const LETTER_A = 65;
```

Comment:
Where we can, would it be better to calculate for clarity? Also maybe it's better to call it `CHAR_A` (broader collection)?

```js
export const CHAR_A = 'A'.charCodeAt(0);
```

Here is some useful info on using `event.keyCode`: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode. Could we use `event.key` instead? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key